_posts/blog/2023-07-01-2023-retreat-okrs.md

```diff
@@ -2,47 +2,62 @@
 layout: blog
 categories: 
   - blog
-title: "2023 OKRs & Board Retreat Highlights"
+title: "Chi Hack Night | 2023-2024 Goals"
 description: ""
-date: 2023-07-10
-image: /images/logo/logo-star.png
+date: 2023-08-22
+image: /images/blog/2023-board-retreat.jpg
 author: Sean Watland
 author_url: https://chihacknight.org/board-of-directors.html
-author_image: /images/logo/logo-star.png
+author_image: /images/people/board/sean_watland.jpg
 author_bio: 
 author2: 
 author2_url: 
 author2_image: 
 author2_bio: 
-published: false
-featured: false
+published: true
+featured: true
 ---
-On June 3th, 2023, the Board of Directors held our first in-person *board retreat*! This retreat served as an excellent opportunity for the 10 Board of Directors to have more focused discussions around *what membership means at Chi Hack Night**, plan our upcoming **OKRs** for the year, and enjoy copious amounts of bagels & coffee. 
+<p class="text-center"><img src="/images/blog/2023-board-retreat.jpg" alt="Picture of our Board of Directions from our board retreat" class='img-responsive' style='width:100%;'/>
+</p>
+
+On June 3th, 2023, the Board of Directors held our first in-person *board retreat*! This retreat served as an excellent opportunity for the 10 Board of Directors to have more focused discussions around what membership means at Chi Hack Night, plan our upcoming **OKRs** for the year, and enjoy copious amounts of bagels & coffee. 
 
 Our retreat was held from 10 am until 3 pm, downtown Chicago at [dscout](www.dscout.com) corporate headquarters. (Thank you [dscout](www.dscout.com) for graciously hosting us for the day!)
 
-# Membership Discussion
+In planning our goals for the upcoming year - three larger goals out of our discussions: 
+1. Continuing to run a high quality civic tech event
+2. Align our membership experiences with our hybrid event
+3. Expand the broader awareness of Chi Hack Night
+
+For the third year, we've used an **OKR** framework to plan and measure our goals. **OKRs** (short for **Objective** & **Key Results**) help us know: 
+1. What are our important focus areas - the **Objectives** 
+2. How do we know we've succeeded - the **Key Results**
+
+*If you are curious to learn more about OKRs and how they work, here's an [article](https://www.whatmatters.com/faqs/okr-meaning-definition-example) that shares more about how they work.*
 
-# OKR Planning
+While you'll see and hear more about the specifics on various ways of going about this, we wanted to share these out so that our wider community can see what we've determined to focus on for the coming year. 
```

Comment:
```suggestion
While you'll see and hear more about the specifics as we work on these throughout the year, we wanted to share these out now so that our wider community can see what we're planning to focus on in the coming months. 
```